src/Pages/MatchPage/style.js

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+
+export const Container = styled.div`
+    height: 100vh;
+    width: 720px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: space-evenly;
+    flex-direction: column;
+    @media screen and (max-width: 720px) {
+        width: 100vw;
+    }
+`;
+
+export const VioletText = styled.h1`
+    position: relative;
+    top: 15vh;
+    font-size: 55px;
+    color: #8f85ff;
+    @media screen and (max-width: 700px) {
+        font-size: 7.3vw;
+    }
+`;
+
+export const NameContainer = styled.form`
+    width: 700px;
+    justify-content: space-evenly;
+    display: flex;
+    align-items: center;
+    @media screen and (max-width: 700px) {
+        flex-direction: column;
+        width: 50vw;
+    }
+`;
+
+export const NameInput = styled.input`
+    outline: none;
+    width: 270px;
+    height: 100px;
+    border: 0;
+    border-bottom: solid 2px ${(props) => props.color || 'white'};
```

Comment:
```suggestion
    border-bottom: solid 2px ${({color}) => color || 'white'};
```

하나 팁을 주자면 props를 가져오는 방식을 구조 분해 할당 (비구조화 할당) 으로도 코드를 작성 할 수 있습니다

Reply:
수정 했습니다
비구조화 할당에 대해서 더 찾아볼게요